src/main/scala/com/gu/scanamo/ScanamoFree.scala

```diff
@@ -1,48 +1,56 @@
 package com.gu.scanamo
 
-import com.amazonaws.services.dynamodbv2.model.{AttributeValue, BatchWriteItemResult, DeleteItemResult, PutItemResult, ScanRequest}
+import com.amazonaws.services.dynamodbv2.model.{PutRequest, WriteRequest, _}
 import cats.data.Xor
 import com.gu.scanamo.DynamoResultStream.{QueryResultStream, ScanResultStream}
 import com.gu.scanamo.error.DynamoReadError
 import com.gu.scanamo.ops.ScanamoOps
 import com.gu.scanamo.query._
-import com.gu.scanamo.request.ScanamoUpdateRequest
+import com.gu.scanamo.request.{ScanamoDeleteRequest, ScanamoPutRequest, ScanamoUpdateRequest}
 import com.gu.scanamo.update.UpdateExpression
 
 object ScanamoFree {
 
-  import Requests._
   import cats.std.list._
   import cats.syntax.traverse._
+  import collection.convert.decorateAsJava._
 
   def given[T: ConditionExpression](tableName: String)(condition: T): ConditionalOperation[T] =
     ConditionalOperation(tableName, condition)
 
   def put[T](tableName: String)(item: T)(implicit f: DynamoFormat[T]): ScanamoOps[PutItemResult] =
-    ScanamoOps.put(putRequest(tableName)(item))
+    ScanamoOps.put(ScanamoPutRequest(tableName, f.write(item), None))
 
-  def putAll[T: DynamoFormat](tableName: String)(items: List[T]): ScanamoOps[List[BatchWriteItemResult]] =
+  def putAll[T](tableName: String)(items: List[T])(implicit f: DynamoFormat[T]): ScanamoOps[List[BatchWriteItemResult]] =
     items.grouped(25).toList.traverseU(batch =>
-      ScanamoOps.batchWrite(batchPutRequest(tableName)(batch)))
+      ScanamoOps.batchWrite(
+        new BatchWriteItemRequest().withRequestItems(Map(tableName -> batch.map(i =>
+          new WriteRequest().withPutRequest(new PutRequest().withItem(f.write(i).getM))
+        ).asJava).asJava)
+      )
+    )
 
   def get[T](tableName: String)(key: UniqueKey[_])
     (implicit ft: DynamoFormat[T]): ScanamoOps[Option[Xor[DynamoReadError, T]]] =
     for {
-      res <- ScanamoOps.get(getRequest(tableName)(key))
+      res <- ScanamoOps.get(new GetItemRequest().withTableName(tableName).withKey(key.asAVMap.asJava))
```

Comment:
So just to be clear; this `new` is because we don't have `GetItemRequest` in `scanamo`... yet. Might be worth eventually adding all the verbs/requests for consistency?